README.md

```diff
@@ -68,6 +68,16 @@ To debug, simply add the parameter:
 
 Now debug remotely by connecting to the port 5005.
 
+### Integration Tests
+By default, Integration Tests are skipped during the build. To run them, you need to follow steps:
+
+First, you need to install the SNAPSHOT version of the root project in the local Maven repository.  
+Then, you need to import the `integationTests` project as a Maven project and ensure that Android SDK is set.  
```

Comment:
I'd suggest to use a bullet list here:
- install the SNAPSHOT ...
- import the `intzegrationTests` ...

Reply:
also should we mention that it may be required to `export ANDROID_SDK_ROOT`?
For example: `export ANDROID_SDK_ROOT=/usr/lib/android-sdk/` or `export ANDROID_SDK_ROOT=/home/<user>/AndroidSDK`

Reply:
Good point, Angelo!
Thank you for your suggestion, both of you. It is updated now.